linker-interface/shared/src/main/scala/org/scalajs/linker/interface/ModuleOutputSpec.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.interface
+
+/** Module output specification. */
+final class ModuleOutputSpec private (modules: Seq[(ModuleOutputSpec.Selector, ModuleOutputSpec.Module)]) {
+  private def this() = this(Nil)
+
+  def addModule(selector: ModuleOutputSpec.Selector, module: ModuleOutputSpec.Module): ModuleOutputSpec = {
```

Comment:
The idea here is that in a future version we can add: `addAutoClassifyModule(selector)` if this feature is needed.